synapse/rest/client/login.py

```diff
@@ -166,12 +166,10 @@ async def on_POST(self, request: SynapseRequest) -> Tuple[int, LoginResponse]:
         if self._msc2918_enabled:
```

Comment:
Is this flag now redundant?

Reply:
I think the flag is worth keeping (does this homeserver use access tokens?). But I interpreted that flag as as "are we running experimental changes for this MSC"?

Suggest renaming it to `self.offers_refresh_tokens`. But maybe you can think of a better name.

Reply:
+1 to renaming it, we tend to use MSC-prefixed enabled flags to mean experimental features!